mkapp/app/script/rc.sh

```diff
@@ -75,6 +75,9 @@ usleep 2000
 #set Microphone Bias Control Register
 aww 0x050967c0 0x110e6100
 
+#dropbear process
+/mnt/app/script/dropbear_start.sh
```

Comment:
lets detach the process here via `&` so blocking portions of the dropbear init (eg. keygen) do not hold up the boot process.

Reply:
Agree will make the change.